virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc

```diff
@@ -0,0 +1,23 @@
+[id="virt-cloning-a-datavolume-using-smart-cloning"]
+= Cloning a DataVolume using smart-cloning
+include::modules/virt-document-attributes.adoc[]
+:context: virt-cloning-a-datavolume-using-smart-cloning
+toc::[]
+
+When you clone a DataVolume (DV), you make a copy of it.
+By using cloning, you reduce the amount of time it takes to deploy many
```

Comment:
Is "using cloning" correct? Could it just be "By cloning, ..." ?

Reply:
Agreed. Using is superfluous.